Core/GDCore/Extensions/Builtin/TimeExtension.cpp

```diff
@@ -135,6 +136,18 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsTimeExtension(
       .AddParameter("expression",
                     _("Scale (1: Default, 2: 2x faster, 0.5: 2x slower...)"));
 
+  extension
+      .AddAction("Wait",
+                 _("Wait X milliseconds"),
```

Comment:
Should we use seconds instead of milliseconds as the other instructions use seconds?

Reply:
Good call